backend/config/curation-api.yml

```diff
@@ -46,32 +53,34 @@ paths:
 
   /v1/collections:
     get:
-      summary: Fetch Collections metadata.
-      description: >-
-        * When the visibility parameter is unspecified or set to `PUBLIC`, a list of all public collections is returned.
-        The Authorization header is not required. If a Collection in the list has been deleted, then it is annotated
-        with tombstone set to True.
+      summary: List Collections' metadata.
+      description: |
+        ListCollections. For each collection in the list, the datasets for that collection include a 
```

Comment:
ListCollections -> List Collections.

Reply:
Also see my earlier [comments](https://github.com/chanzuckerberg/single-cell-data-portal/issues/2161#issuecomment-1232243125).

Reply:
Also need to capitalize `Collection` and `Dataset` in this description